docs/metric_group_deye_sg01hp3.md

```diff
@@ -1,6 +1,8 @@
 |Metric|MQTT topic suffix|Unit|Modbus address (dec)|Modbus address (hex)|Modbus data type|Scale factor|
 |---|---|:-:|:-:|:-:|:-:|:-:|
 |Running status|`inverter/status`|N/A (0: standby, 1: selfcheck, 2: normal, 3: alarm, 4: fault)|500|1f4|U_WORD|1|
+|AC relays status|`ac/relay_status`|bit array supposedly storing states of AC relays|552|228|U_WORD|1|
+|On-grid|`ac/ongrid`|True/False|552 (bit no. 2)|228 (bit no. 2)|n/a|1|
```

Comment:
Documentation files are generated from the source code, these changes will be overwritten.